apps/frontend/web/src/pages/AccountSettings.tsx

```diff
@@ -122,26 +127,34 @@ const AccountSettings = () => {
           <UserImageFiller
             selectedImage={selectedImage}
             setSelectedImage={setSelectedImage}
+            formData={formData}
+            setFormData={setFormData}
           />
         }
         submitText="Save"
-        submitFunction={() => setGamerImage(selectedImage)}
+        action={getFormData()}
```

Comment:
```suggestion
        action={getFormData}
```
comme pour le `setFormData` 4 lignes au dessus